### DISCUSSION TITLE
How to disable workers "globally"?

### DISCUSSION THREAD

**Discussion by info-sic** [pandora-analysis/pandora#522] [Q&A] [answered]
If I don't want my users to use e.g. Lookyloo, and it should not appear in the "advanced" list on submit.html how do I do it?
replicas 0 in lookyloo.yml? Delete lookyloo.* from pandora/workers? I want it to be persistent, even after an `poetry run update --yes`.
Also: is there a way to set a standard for which workers will be "enabled" (ticked ON) in the "advanced" list on submit.html?
I still don't understand, why, how and which workers do or don't do their job :)

**Comment by Rafiot** [comment, marked as answer]
the easiest way is to do the following:

1. remove the .yml file
2. run `update` without --yes

If you want to keep running update with --yes, remove the `<module>.py` file and make a commit.

**Reply by info-sic** [reply]
Thx
